### PR TITLE
.github: update release process

### PR DESCRIPTION
**Pull request by aanm**
Add steps to sync projects during release process.

Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

**Review by christarazi — APPROVED** [review]
:tada:

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 1]
```diff
@@ -26,17 +25,18 @@ assignees: ''
...
   - [ ] Pull latest branch
-  - [ ] Run `contrib/release/start-release.sh'
+  - [ ] Run `contrib/release/start-release.sh`
   - [ ] (If applicable) Update the `cilium_version` and `cilium_tag` in
         `examples/getting-started/Vagrantfile`
   - [ ] Run `Documentation/check-crd-compat-table.sh vX.Y` and if needed, follow the
         instructions.
   - [ ] Commit all changes with title `Prepare for release vX.Y.Z`
   - [ ] Submit PR (`contrib/release/submit-release.sh`)
+- [ ] Execute `release --current-version X.Y.Z --next-dev-version X.Y.W` to automatically
```
Comment (nebril) [inline .github/ISSUE_TEMPLATE/release_template.md:37]: is `release` a custom binary or makefile target?

Comment (aanm) [reply .github/ISSUE_TEMPLATE/release_template.md:37]: it's a binary of github.com/cilium/releases

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 2]
```diff
@@ -26,17 +25,18 @@ assignees: ''
...
+      move any unresolved issues/PRs from old release project into the new
+      project. (`W` should be calculation of `Z+1`)
```
Comment (joestringer) [inline .github/ISSUE_TEMPLATE/release_template.md:39]: Would be nice to make this part of the start-release step so it's one less moving piece to think about.

Comment (joestringer) [reply .github/ISSUE_TEMPLATE/release_template.md:39]: Not sure if we have the number `z+1` in the current scripts though, that can be non-trivial to calculate in shell scripts :<

**Pull request merged**